index.html

```diff
@@ -1,185 +1,221 @@
-                                                        // remove word from corrected words
-                                                        this.correctedWords = this.correctedWords.filter(correctedWord => correctedWord.correct !== word.correct)
-                                                  
-                                                        console.log(this.richText);
-                                                },
-                                                onInput(event) {
-                                                        const value = event.target.innerText;
-                                                        this.event = event
-                                                        this.text = value;
-                                                },
-                                        },
-                                        watch: {}  
-                                }) 								
-                                
-							 
-				
-							
-                        </script>
-
-        </body>
-
-        <!--End of page -->
-
-</html>
+<!DOCTYPE html/>
+<html lang=en>
+        <!--The website header contents and links-->
+        <head>
+                <title>Group4spellchecker</title>
+				<meta charset="UTF-8">
+                <meta name="viewport" content="width=device-width, initial-scale=1.0">
+                <link rel="stylesheet" href="index.css">
+                <script src="https://cdn.jsdelivr.net/npm/vue@2.6.14/dist/vue.js"></script>
+                <script src="https://cdn.jsdelivr.net/npm/axios@0.12.0/dist/axios.min.js"></script>
+                <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        </head>
+
+        <!-- body of the HTML content -->
+        <body>
+                <!--div to allow manipulation of the content and suggestion area using Vue JavaScript framework-->
+                <div id="app" class="bg-light">
+
+                        <!-- The content header -->
+                        <div class="container px-4 pt-4">
+                                <div class="d-flex align-items-center p-3 my-3 text-white bg-blue rounded shadow-sm">
+                                        <div class="lh-1 p-3">
+                                                <h1 class="h3 mb-0 text-white lh-1">G4 Spell Checker Machine</h1>
+                                        </div>
+                                </div>
+                        <!-- The menu to use to manipulate the HTML page -->
+                        <div class="row g-3 py-3 pt-5 row-cols-1 row-cols-lg-2">
+                                <div class="col">
+                                        <button id='clr' type="button" class="btn  btn-sm btn-outline-primary" onclick="getElementById('mainContentBox').innerHTML = ''" >Clear</button>

```

Comment:
@MarshallOkafor This is were Clear button is programed and we don't have to write a separate function about same.

Reply:
@Bsharma4 Great! I inadvertently skipped the label of the button. Thanks for letting me know. I have also seen in it when I updated and run the code on my system.

Well done guys!